eudemo/eudemo_tests/test_maintenance.py

```diff
@@ -52,3 +63,96 @@ def test_dummy_blanket_port_opt(self):
         solution = design_problem.optimise()
 
         assert design_problem.opt.check_constraints(solution)
+
+
+class TestEquatorialPortDesigner:
+    """Tests the Equatorial Port Designer"""
+
+    def setup_method(self) -> None:
+        """Set-up Equatorial Port Designer"""
+        params = {
+            "ep_height": {"value": 0, "unit": "m"},
+        }
+
+        self.designer = EquatorialPortDesigner(params, None, 0, 0)
+
+    @pytest.mark.parametrize(
+        "xi, xo, zh", zip([2.0, 3.0, 1.0], [9.0, 9.0, 4.0], [5.0, 4.0, 2.0])
+    )
+    def test_ep_designer(self, xi, xo, zh):
+        """Test Equatorial Port Designer"""
+        param_values = {"ep_height": zh}
+        self.designer.params.update_values(param_values)
+        self.designer.x_ib = xi
+        self.designer.x_ob = xo
+        output = self.designer.execute()
+
+        # x = (xi, xo, xo, xi)
+        # z = (-zh / 2, -zh / 2, zh / 2, zh / 2)
+        # expectation = BluemiraFace(make_polygon({"x": x, "y": 0, "z": z}, closed=True))
+
+        # assert math.isclose(output.length, expectation.length)
+        # assert math.isclose(output.area, expectation.area)
+        # assert math.isclose(output.volume, expectation.volume)
+
+        x_len = xo - xi
+        z_len = zh
+
+        try:
+            assert output.length == 2 * (x_len + z_len)
+        except AssertionError:
+            assert math.isclose(output.length, 2 * (x_len + z_len))
+        try:
+            assert output.area == x_len * z_len
+        except AssertionError:
+            math.isclose(output.area, x_len * z_len)
```

Comment:
Ahh I meant here that the comment probably wasnt needed and could be removed. I would leave the isclose checks because they.re more likely to fail than not.

```suggestion
        assert math.isclose(output.length, 2 * (x_len + z_len))
        assert math.isclose(output.area, x_len * z_len)
```